.eslintrc

```diff
@@ -14,7 +14,7 @@
       "plugin:@typescript-eslint/recommended"
```

Comment:
We forgot to reference the ESLint plugin:
https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#usage